docs/flutter/v2/guides/quickstart.mdx

```diff
@@ -5,316 +5,868 @@ nav: 2.0
-
-  ///when network or some other error happens it will be called
-  void onReconnecting();
-
-  ///when you are back in the room after reconnection
-  void onReconnected();
-
-  ///when someone requests for track change of yours be it video or audio this will be triggered
-  /// [hmsTrackChangeRequest] request instance consisting of all the required info about track change
-  void onChangeTrackStateRequest({required HMSTrackChangeRequest hmsTrackChangeRequest});
-
-  ///when someone kicks you out or when someone ends the room at that time it is triggered
-  /// [hmsPeerRemovedFromPeer] it consists info about who removed you and why.
-  void onRemovedFromRoom({required HMSPeerRemovedFromPeer hmsPeerRemovedFromPeer});
-
-  ///whenever a new audio device is plugged in or audio output is changed we
-  ///get the onAudioDeviceChanged update
-  ///This callback is only fired on Android devices. On iOS, this callback will not be triggered.
-  /// - Parameters:
-  ///   - currentAudioDevice: Current audio output route
-  ///   - availableAudioDevice: List of available audio output devices
-  void onAudioDeviceChanged(
-      {HMSAudioDevice? currentAudioDevice,
-      List<HMSAudioDevice>? availableAudioDevice});
-}
+    <br />
 
-```
+    ```xml
+    <key>NSMicrophoneUsageDescription</key>
+    <string>{YourAppName} wants to use your microphone</string>
 
-### How you know when people join or leave
+    <key>NSCameraUsageDescription</key>
+    <string>{YourAppName} wants to use your camera</string>
 
-The join method takes an interface called `HMSUpdateListener`. It lets you know when peers join and leave the call, mute/unmute their audio and video and lots more.
+    <key>NSLocalNetworkUsageDescription</key>
+    <string>{YourAppName} App wants to use your local network</string>
+    ```
 
-The `HMSUpdateListener` has a callback to notify about people joining or leaving. It is `onPeerUpdate(type: HMSPeerUpdate, peer: HMSPeer)`.
+2. Add the target platform version as (`platform :ios, '12.0'`) in "ios/Podfile."
 
-> 💡 HMSPeer is object that represents a person in the call.
+#### Build and run the app
 
----
+-   Once you've made the above changes, your app is ready for testing. You can build the app and run it in a simulator or an actual iOS device.
+-   Go to _Run > Start debugging > select a device to use_ (iOS simulator or iPhone connected via USB).
 
-### How to render audio and video
+Now, after you click `join`, you should be able to see yourself (the android emulator doesn't support actual video, you can connect an actual device to see your video in real-time). You can join the room using a browser as the second peer to check audio/video transactions between two or more peers.
 
-The SDK plays the audio for every person who joins the call. Audio will begin playing when join succeeds.
-To render the person's video you need to create an instance of HMSVideoView.
+<video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
```

Comment:
Same as above here. iOS Simulator does not publish any video, Android Emulator at least shows a Bitmap gif sort of thing.
It would be better if the video showed running on Android/iPhone devices instead of Emulators. 
Since emulators do not publish video, first-time users get the impression that 100ms is not working.

Reply:
@ygit please help me with the screenrecorded video for iOS. Thanks!